improver/cli/threshold.py

```diff
@@ -129,15 +127,15 @@ def process(
         thresholds = []
         fuzzy_bounds = []
         for key in threshold_config.keys():
-            thresholds.append(np.float32(key))
+            thresholds.append(float(key))
```

Comment:
A comment is probably needed here, otherwise this change could easily be reverted by someone in a well intentioned way. What about `Ensure thresholds are float64 to avoid rounding errors during possible unit conversion.`?

Reply:
Good point!  Thanks @gavinevans